fastfield_codecs/src/column.rs

```diff
@@ -165,6 +173,23 @@ where
     fn num_vals(&self) -> u64 {
         self.from_column.num_vals()
     }
+
+    fn iter<'a>(&'a self) -> Box<dyn Iterator<Item = Output> + 'a> {
+        Box::new(self.from_column.iter().map(&self.monotonic_mapping))
+    }
+
+    fn get_range(&self, mut start: u64, output: &mut [Output]) {
+        let mut intermediary: MaybeUninit<[Input; 16]> = MaybeUninit::uninit();
+        let intermediary_ref = unsafe { intermediary.assume_init_mut() };
```

Comment:
I would prefer default initialization with a constraint on `Input: Default` . This should probably be optional, if output is >= 16 ?